doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+
+1. Go to `nRF Cloud`_ and sign in.
+#. Click :guilabel:`Device Management` in the left pane and select :guilabel:`Provision Devices`.
+
+   .. figure:: /images/nrfcloud_provisiondevices.png
+      :alt: nRF Cloud - Provision Devices
+
+      nRF Cloud - Provision Devices
+
+#. In the  page that opens up, click on the gear icon near the upper right corner and select :guilabel:`Create JITP Certificates`.
+
+   .. figure:: /images/nrfcloud_provisiondevices_gearicon.png
+      :alt: nRF Cloud - Gear icon in Provision Devices
+
+      nRF Cloud - Gear icon in Provision Devices
+
+   The :guilabel:`Create JITP Certificates` dialog box opens.
+
+   .. figure:: /images/nrfcloud_jitpcertificates.png
+      :alt: nRF Cloud - Create JITP Certificates dialog box
+
+      nRF Cloud - Create JITP Certificates dialog box
+
+#. Enter the following information:
+
+   * **Device ID:** the device ID is composed of *nrf-* and the 15-digit :term:`International Mobile (Station) Equipment Identity (IMEI)` number that is printed on the label of your nRF9160 DK.
+     For example, *nrf-123456789012345*.
+   * **Owndership code:** the ownership code is the PIN or the hardware ID of your DK, and it is found on the label of your nRF9160 DK.
+
+     If the label contains a PIN in addition to the IMEI number, enter this pin.
+     If it does not contain a PIN, enter the Hardware ID (HWID) HEX code, with or without colons.
+     For example, *AA:BB:CC:DD:EE:FF* or *AABBCCDDEEFF*.
+
+     .. note::
+
+        The ownership code serves as a password and proves that you own the specific nRF9160 DK.
+        Therefore, you should not share it with anyone.
+
+#. Click :guilabel:`Download Certificate` and save the :file:`.cert.json` file to a folder of your choice.
+
+   .. note::
+
+      The certificate contains all the information that is needed to connect your nRF9160 DK to nRF Cloud.
+      Therefore, you should not share it with anyone.
+
+.. _provisioning_cloud_certificate:
+
+Provisioning the nRF Cloud certificate
+======================================
+
+After downloading the certificate, you must provision it to your nRF9160 DK.
+
+.. note::
+
+   The application firmware on the nRF9160 DK must support long AT commands up to 3kB to provision the certificate.
+   If you have :ref:`updated the application firmware <nrf9160_gs_updating_fw_application>`, this requirement is fulfilled.
+
+Complete the following steps to provision the certificate:
+
+1. Open the LTE Link Monitor app from nRF Connect for Desktop.
+#. In the :guilabel:`Settings` pane of the side panel, deselect the checkbox for :guilabel:`Automatic requests` if it is selected.
+
+   .. figure:: /images/ltelinkmonitor_automaticrequests.png
+      :alt: LTE Link Monitor - Automatic requests check box
+
+#. If you have already inserted the SIM card into your DK, remove it before you continue.
+#. Connect the nRF9160 DK to the computer with a micro-USB cable, and turn it on.
```

Comment:
Sometimes the DK is referred to as "DK", sometimes "nRF9160 DK". Suggest to use one consistently.

Reply:
Tried to keep to starting each section with "nRF9160 DK" and using the full name in notes, and the rest of the mentions as "DK" within the section. Missed one instance here, fixed.